Sources/Model/Receipt/HyperwalletReceipt.swift

```diff
@@ -127,109 +125,204 @@ public struct HyperwalletReceipt: Decodable, Equatable {
         case manualAdjustment = "MANUAL_ADJUSTMENT"
+        /// The payment expiration
         case paymentExpiration = "PAYMENT_EXPIRATION"
-        // Bank account-specific types
+        /// Bank account-specific types
+        ///
+        /// The bank account transfer fee
         case bankAccountTransferFee = "BANK_ACCOUNT_TRANSFER_FEE"
+        /// The bank account transfer return
         case bankAccountTransferReturn = "BANK_ACCOUNT_TRANSFER_RETURN"
+        /// The bank account transfer return fee
         case bankAccountTransferReturnFee = "BANK_ACCOUNT_TRANSFER_RETURN_FEE"
+        ///
         case transferToBankAccount = "TRANSFER_TO_BANK_ACCOUNT"
-        // Card-specific types
+        /// Card-specific types
+        ///
+        /// The card activation fee
         case cardActivationFee = "CARD_ACTIVATION_FEE"
+        /// The card activation fee waiver
         case cardActivationFeeWaiver = "CARD_ACTIVATION_FEE_WAIVER"
+        /// The card fee
         case cardFee = "CARD_FEE"
+        ///
         case manualTransferToPrepaidCard = "MANUAL_TRANSFER_TO_PREPAID_CARD"
+        /// The prepaid card balance inquiry fee
         case prepaidCardBalanceInquiryFee = "PREPAID_CARD_BALANCE_INQUIRY_FEE"
+        /// The prepaid card cash advance
         case prepaidCardCashAdvance = "PREPAID_CARD_CASH_ADVANCE"
+        /// The prepaid card disputed charge refund
         case prepaidCardDisputedChargeRefund = "PREPAID_CARD_DISPUTED_CHARGE_REFUND"
+        /// The prepaid card dispute deposit
         case prepaidCardDisputeDeposit = "PREPAID_CARD_DISPUTE_DEPOSIT"
+        /// The prepaid card domestic cash withdrawal fee
         case prepaidCardDomesticCashWithdrawalFee = "PREPAID_CARD_DOMESTIC_CASH_WITHDRAWAL_FEE"
+        /// The prepaid card exchange rate difference
         case prepaidCardExchangeRateDifference = "PREPAID_CARD_EXCHANGE_RATE_DIFFERENCE"
+        /// The prepaid card manual unload
         case prepaidCardManualUnload = "PREPAID_CARD_MANUAL_UNLOAD"
+        /// The prepaid card overseas cash withdrawal fee
         case prepaidCardOverseasCashWithdrawalFee = "PREPAID_CARD_OVERSEAS_CASH_WITHDRAWAL_FEE"
+        /// The prepaid card pin change fee
         case prepaidCardPinChangeFee = "PREPAID_CARD_PIN_CHANGE_FEE"
+        /// The prepaid card refund
         case prepaidCardRefund = "PREPAID_CARD_REFUND"
+        /// The prepaid card replacement fee
         case prepaidCardReplacementFee = "PREPAID_CARD_REPLACEMENT_FEE"
+        /// The prepaid card sale reversal
         case prepaidCardSaleReversal = "PREPAID_CARD_SALE_REVERSAL"
+        /// The prepaid card unload
         case prepaidCardUnload = "PREPAID_CARD_UNLOAD"
+        /// The transfer to debitcard
         case transferToDebitCard = "TRANSFER_TO_DEBIT_CARD"
+        /// The transfer to prepaid card
         case transferToPrepaidCard = "TRANSFER_TO_PREPAID_CARD"
-        // Donation types
+        /// Donation types
+        ///
```

Comment:
No need to add this in document so just // are fine and remove the extra ///

Reply:
Removed extra /